lib/db/files_db.dart

```diff
@@ -1254,6 +1255,28 @@ class FilesDB {
     return result;
   }
 
+  // getCollectionLatestFileTime returns map of collectionID to the max
+  // creationTime of the files in the collection.
+  Future<Map<int, int>> getCollectionIDToMaxCreationTime() async {
```

Comment:
We will also need to cache this result to provide sort by newest photo functionality. It's still much faster than `getLatestCollectionFiles`